src/DynamoRevitIcons/DynamoRevitIcons.csproj

```diff
@@ -34,7 +34,9 @@
   </PropertyGroup>
   <ItemGroup>
     <EmbeddedResource Include="DSRevitNodesUIImages.resx" />
-    <EmbeddedResource Include="RevitNodesImages.resx" />
+    <EmbeddedResource Include="RevitNodesImages.resx">
```

Comment:
Is this not needed in the new PR? The number of files changed seem to be different for the 2 PR's. Can you please cross check that.